packages/split-button/test/index.ts

```diff
@@ -97,6 +98,73 @@ export function runSplitButtonTests(
         await expect(el1).to.be.accessible();
         await expect(el2).to.be.accessible();
     });
+    it('loads splitbutton accessibly and checks labels', async () => {
+        const test = await fixture<HTMLDivElement>(html`
+            <div>
+                <sp-split-button>${deprecatedMenu()}</sp-split-button>
+                <sp-split-button label="Test" left>
+                    ${deprecatedMenu()}
+                </sp-split-button>
+            </div>
+        `);
+        const el1 = test.querySelector('sp-split-button') as SplitButton;
+        const el2 = test.querySelector('sp-split-button[left]') as SplitButton;
+
+        await elementUpdated(el1);
+        await elementUpdated(el2);
+
+        type NamedRoledPopupNode = {
+            name: string;
+            role: string;
+            haspopup: boolean;
+        };
+        const snapshot = (await a11ySnapshot(
+            {}
+        )) as unknown as NamedRoledPopupNode & {
+            children: NamedRoledPopupNode[];
+        };
+        expect(
+            findAccessibilityNode<NamedRoledPopupNode>(
+                snapshot,
+                (node) =>
+                    (node.role === 'button' || node.role === 'buttonmenu') &&
```

Comment:
@jnurthen this gets the test passing in Firefox. Not sure if this counts as a _bug_ in Firefox (or possibly in Playwright, they're deprecating these API anyways 😢), but it seems to be OK on our end.

If it's good for you, I'm good to merge this in.

Reply:
thanks for fixing that. That looks good.